### DISCUSSION TITLE
The "fsthick" in electron-phonon coupling calculation for metal

### DISCUSSION THREAD

**Discussion by zengyoga** [nakib/elphbolt#168] [General]
Dear nakib,

I try to reproduce the results of electron and phonon thermal conductivity of BCC tungsten [Phys. Rev. B, 99, 020305, 2019] . However, I find that the results are strongly determined by the parameter `fsthick` in `elphbolt`

I test this parameter from 0.05 ~ 0.40, however, the lattice and electron thermal conductivity are still changed significantly.

To fit the experimental results, I found the "fsthick = 0.11" for tungsten is the best choice at 200K.

However, if such value is also used in the calcuation at T = 300K, I will get the same results as T = 200K.

For this reason, I currently change this parameter by the following rules:

1. Following the energy equipartition theorem, I estimate the energy at corresponding temperature.

2. Then, 4 times of this energy is set to `fsthick`

Could you give me some suggestions for this parameter setting? My `input.nml` is given in following:

Thanks!

```
&allocations
        numelements=1
        numatoms=1
/

&crystal_info
	name               = 'BCC W'        !name of material
	elements           = "W"            !elements in the basis
        atomtypes          = 1              !integer tagging unique elements in the basis
        masses             = 183.7400       !masses of basis atom in amu. (if it not provided, set autoisotopes=.true.)
        lattvecs(:,1)      = -0.1588848304   0.1588848304   0.1588848304
        lattvecs(:,2)      =  0.1588848304  -0.1588848304   0.1588848304
        lattvecs(:,3)      =  0.1588848304   0.1588848304  -0.1588848304
        basis(:,1)         =  0.5000000000   0.5000000000   0.5000000000
	T                  = 200.0          !temperature in K
/

&electrons
	spindeg            = 2              !spin degeneracy of the bands
	enref              = 22.2905        !Electron reference energy in eV (center of transport active window)
	chempot            = 22.2905        !Chemical potential in eV
	metallic           = .true.         !Is the system metallic
	numbands           = 9              !Total wannier bands
	indlowband         = 3              !Lowest transport band
	indhighband        = 5              !Highest transport band
/

&numerics
	qmesh            = 50 50 50        !phonon wave vector mesh(q)
        mesh_ref         = 3               !qmesh*mesh_ref = emesh
	fsthick          = 0.11           !eV about enref
	datadumpdir      = './scratch/'    !Or, enter suitable scratch directory
        phe              = .true.          !include phonon-electron interaction in phonon BTE
        phiso            = .true.          !include phonon-isotopes interaction in phonon BTE
        drag             = .true.          !include electron and phonon drag term in the phonon and electron BTE.
        maxiter          = 100             !maximum number of iteration steps for the BTE(s)
        conv_thres       = 1e-3            !relative convergence threshold for the BTE(s)
        runlevel         = 1               !control for type of calculation. (details see manual)
/

&wannier
        coarse_qmesh      = 6 6 6          !coarse phonon wave vector mesh employed in wannier calculation. It must be match with ph.x calculation
/

**Comment by nakib** [comment]
Dear @zengyoga,

Thank you for your query. Would you be able to share some numbers so that I can see how much your transport coefficients are varying with respect to the `fsthick` parameter?

A good rule of thumb for choosing this parameter would be to take either the width of f0(1 - f0) for the highest temperature you are going to use or some factor times the highest phonon energy, whichever is the larger energy scale. From the materials that I have calculated so far, 0.4 eV - 0.5 eV should suffice, unless you are going to temperatures beyond 500 K. 0.11 eV seems too low for 300 K.

Best,
Nakib

**Reply by zengyoga** [reply]
Dear nakib,

Thank you for your reply. I will send a summary of the results here after the test.

Thanks

**Reply by nakib** [reply]
Thanks. Another point: for 3D cases, consider using the tetrahedron method.

Best,
Nakib
